### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Muhamedeyada**

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -9,70 +8,104 @@ import {
...
-          rel="noopener noreferrer"
-          className="footer-social-link"
-        >
-          <FaYoutube />
-        </a>
+    <footer className="bg-[#a5c926] pt-12 text-white">
+      <div className="container mx-auto flex flex-col lg:flex-row justify-between items-center space-y-6 lg:space-y-0">
+        {/* Footer Left - Brand */}
+        <div className="footer-left">
+          <a href="/" className="text-2xl font-bold">
```
Comment (ZiadNouh) [inline src/components/Footer.jsx:15]: use Link element for components and pages inside our app

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -9,70 +8,104 @@ import {
...
+            ToyStore
+          </a>
+        </div>
+
+        <div className="footer-nav flex space-x-6">
+          <a href="/" className="footer-link hover:underline">
```
Comment (ZiadNouh) [inline src/components/Footer.jsx:21]: use Link element for components and pages inside our app

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -9,70 +8,104 @@ import {
...
+            Home
+          </a>
+          <a href="/catalog" className="footer-link hover:underline">
```
Comment (ZiadNouh) [inline src/components/Footer.jsx:24]: use Link element for components and pages inside our app

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -9,70 +8,104 @@ import {
...
+            Catalog
+          </a>
+          <a href="/delivery" className="footer-link hover:underline">
```
Comment (ZiadNouh) [inline src/components/Footer.jsx:27]: use Link element for components and pages inside our app

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -9,70 +8,104 @@ import {
...
+            Delivery
+          </a>
+          <a href="/about" className="footer-link hover:underline">
```
Comment (ZiadNouh) [inline src/components/Footer.jsx:30]: use Link element for components and pages inside our app

File: src/components/Footer.jsx [hunk 6]
```diff
@@ -9,70 +8,104 @@ import {
...
+            About
+          </a>
+          <a href="/contacts" className="footer-link hover:underline">
```
Comment (ZiadNouh) [inline src/components/Footer.jsx:33]: use Link element for components and pages inside our app

File: src/pages/NotFound.jsx [hunk 7]
```diff
@@ -1,9 +1,23 @@
...
+    <div
+      className="flex items-center justify-center min-h-screen bg-cover bg-center"
+      style={{ backgroundImage: `url(${CoverImg})` }}
+    >
+      <div className="bg-white p-6 rounded-lg shadow-lg text-center">
+        <h1 className="text-2xl font-bold mb-3">Page Not Found</h1>
+        <p className="mb-4">
+          The page you are looking for doesnt exist or has been moved.
+        </p>
+        <a href="/" className="btn bg-[#a5c926]">
```
Comment (ZiadNouh) [inline src/pages/NotFound.jsx:15]: use Link element for components and pages inside our app

**Review by AHmedEMadEG — APPROVED** [review]

**Review by ZiadNouh — CHANGES_REQUESTED** [review]

**Pull request merged**